### PR TITLE
chore(expenses): claudio wunder - collab summit

### PR DESCRIPTION
**Pull request by ovflowd**
**Request for:** Collab Summit
**Dates:** 30th September - 3rd October
**Travelling from:** Munich
**Flight:** ~430 USD
**Hotel:** ~670 USD
**Total:** ~1110 USD

I couldn't get funding from my Employer, as they're holding unnecessary costs for now due to the Economic recession.

The member will pay all other costs like Transportation and Food. No expensing is needed.

Signed-off-by: Claudio Wunder <cwunder@gnome.org>

### CODE REVIEW

**Review by bnb — APPROVED** [review]

**Review by Trott — APPROVED** [review]

File: TRAVEL_FUND/2022.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ Sutherland | Ian | OpenJS World 2022 | Project Member || Austin, TX | 06 - 10 Ju
 Lea | Verou | TAG meeting | TAG member || London, UK | 25 - 28 Jul 2022 | 1800 USD | 10 Jul 2022 | tbd
 Erick | Wendel | NodeConf EU | Project Member || Ireland, IE | 02 - 08 Octo 2022 | 2953.59 USD | 22 Aug 2022 | [link](https://github.com/openjs-foundation/cross-project-council/pull/926)
 Waleed | Ashraf | Collab Summit | CPC || Ireland | 30 Sept - 6 Oct 2022 | 1825 USD | 30 Aug 2022 | [link](https://github.com/openjs-foundation/cross-project-council/pull/931)
-||||||| Total | 20,859.59 USD |
+Claudio | Wunder | Collab Summit | Project Member || Ireland | 30 Sept - 3 Oct 2022 | 1825 USD | 10 Sept 2022 | tbd
```
Comment (WaleedAshraf) [inline TRAVEL_FUND/2022.md:16]:
```suggestion
Claudio | Wunder | Collab Summit | Project Member || Ireland | 30 Sept - 3 Oct 2022 | 1050 USD | 10 Sept 2022 | tbd
||||||| Total | 21,909.59 USD |
```

**Review by WaleedAshraf — CHANGES_REQUESTED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

File: TRAVEL_FUND/2022.md (hunk 1 above)
Comment (ovflowd) [reply TRAVEL_FUND/2022.md]: Tru! Forgot to update the text there 🙈

**Comment by ovflowd** [comment]
@WaleedAshraf I updated the doc 🙌

**Review by WaleedAshraf — APPROVED** [review]

**Review by tobie — APPROVED** [review]

**Review by dylans — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by ovflowd** [comment]
Hey folks 👋 from the day I created this issue to today, the flight price increased to 430 USD. Would this price still be OK? (I'm going to book the flight now regardless, so if the new price is not okay, I just expense the original price).

**Comment by ovflowd** [comment]
cc @tobie @mcollina @dylans @bnb @Trott @WaleedAshraf @mhdawson (need a re-approval)

**Review by ljharb — APPROVED** [review]

**Comment by WaleedAshraf** [comment]
@ovflowd you can merge this as it has got enough approvals.
When you are going to expense it after the event, you can create a new PR to update the amount with the actual price.

**Comment by bnb** [comment]
> 
Hey folks 👋 from the day I created this issue to today, the flight price increased to 430 USD. Would this price still be OK? (I'm going to book the flight now regardless, so if the new price is not okay, I just expense the original price).

IMO yes, that's fine.

**Pull request merged**